src/migrations/20190829130939-create-department.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Departments', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    department: {
+      type: Sequelize.STRING
+    },
+    manager: {
```

Comment:
And why do we have manager field, also why is it tied to `users` table?

Reply:
A department has a manager and a manager is a user

Reply:
We should have a `Role table` instead of `Department`